templates/docker-compose.yml.j2

```diff
@@ -6,7 +6,9 @@ services:
     restart: always
     ports:
       - '{{ oauth_local_addr }}:{{ oauth_local_port }}:{{ oauth_local_port | mandatory }}/tcp'
-    extra_hosts: {{ oauth_cont_etc_hosts | to_json }}
+{% if not oauth_nested_compose %}
+  extra_hosts: {{ oauth_cont_etc_hosts | to_json }}
```

Comment:
I think this indentation is wrong. Besides that, looks good!

Reply:
Why do we even need this? Couldn't we just set `oauth_cont_etc_hosts` to an empty list?